builtin/mainmenu/tab_online.lua

```diff
@@ -42,20 +42,26 @@ local function get_formspec(tabview, name, tabdata)
 	local search_panel
 	if mobile then
 		search_panel =
-			"field[0.2,0.1;5.1,1;Dte_search;;" .. esc(tabdata.search_for) .. "]" ..
-			"image_button[4.87,-0.13;0.83,0.83;" .. defaulttexturedir ..
+			"formspec_version[3]" ..
+			"image[-0.1,4.9;6.05,0.89;" .. defaulttexturedir .. "desc_bg.png;32]" ..
+			"style[Dte_search;border=false;bgcolor=transparent]" ..
+			"field[0.25,5.2;4.97,1;Dte_search;;" .. esc(tabdata.search_for) .. "]" ..
+			"image_button[4.85,4.93;0.83,0.83;" .. defaulttexturedir ..
 				"search.png;btn_mp_search;;true;false]" ..
-			"image_button[5.62,-0.13;0.83,0.83;" .. defaulttexturedir ..
+			"image_button[5.6,4.93;0.83,0.83;" .. defaulttexturedir ..
 				"refresh.png;btn_mp_refresh;;true;false]" ..
-			"image_button[6.37,-0.13;0.83,0.83;" .. defaulttexturedir ..
+			"image_button[6.35,4.93;0.83,0.83;" .. defaulttexturedir ..
 				(serverlistmgr.mobile_only and "online_mobile" or "online_pc") .. ".png" ..
 				";btn_mp_mobile;;true;false]"
 	else
 		search_panel =
-			"field[0.2,0.1;5.8,1;Dte_search;;" .. esc(tabdata.search_for) .. "]" ..
-			"image_button[5.62,-0.13;0.83,0.83;" .. defaulttexturedir ..
+			"formspec_version[3]" ..
```

Comment:
I don't think the `formspec_version[3]` here does anything since the `size[]` is added before it.

Reply:
The field will become under the image without this

Reply:
Apparently `formspec_version` works partway through formspecs

Reply:
Apparently `formspec_version` works partway through formspecs